### PR TITLE
fix(deletion): typo fix dataset_bucket -> datasets_bucket

### PR DESCRIPTION
**Pull request by Bento007**

### CODE REVIEW

**Review by prathapsridharan — APPROVED** [review]
lgtm

File: .happy/terraform/modules/lambda/main.tf [hunk 1]
```diff
@@ -10,7 +10,7 @@ resource aws_lambda_function lambda_job_def {
     variables = {
       ARTIFACT_BUCKET            = var.artifact_bucket,
       CELLXGENE_BUCKET           = var.cellxgene_bucket,
-      DATASETS_BUCKET            = var.dataset_bucket
+      DATASETS_BUCKET            = var.datasets_bucket
```
Comment (nayib-jose-gloria) [inline .happy/terraform/modules/lambda/main.tf:13]: DATASETS_BUCKET -> DATASET_BUCKET, as referenced [here](https://github.com/chanzuckerberg/single-cell-data-portal/commit/8f2ab483aea7dcfea983dceeccb8bfe9e1575ff2#diff-f59992ec0be45312f0ebeeb2c6410949e04d8f3ccc9815ff028868ca75b90196)

**Review by nayib-jose-gloria — APPROVED** [review]

File: tests/unit/processing/test_handle_error.py [hunk 2]
```diff
@@ -374,7 +374,11 @@ def test_get_failure_slack_notification_message_with_dataset_and_collection(
 
 @pytest.fixture
 def mock_env_vars():
-    return {"ARTIFACT_BUCKET": "artifact_bucket", "DATASET_BUCKET": "dataset_bucket", "CELLXGENE_BUCKET": "cxg_bucket"}
+    return {
+        "ARTIFACT_BUCKET": "artifact_bucket",
+        "DATASETS_BUCKETS": "datasets_bucket",
```
Comment (nayib-jose-gloria) [inline tests/unit/processing/test_handle_error.py:379]: DATASET_BUCKETS -> DATASET_BUCKET

Comment (nayib-jose-gloria) [reply tests/unit/processing/test_handle_error.py:379]:
```suggestion
        "DATASETS_BUCKET": "datasets_bucket",
```

**Pull request merged**